website/docs/configuration/resources.html.md

```diff
@@ -396,16 +417,19 @@ as a whole.
 #### Using Sets
 
 The Terraform language doesn't have a literal syntax for
-[sets](./types.html#collection-types), but you can use the `toset` function to
-convert a list of strings to a set:
+[set values](./types.html#collection-types), but you can use the `toset`
+function to explicitly convert a list of strings to a set:
 
 ```hcl
-variable "subnet_ids" {
-  type = list(string)
```

Comment:
Great change, since variables are a case where one could define the type as `set(string)` and move on with your day. Nice improvement! (aaaand I see you've noted that here in the docs below! nice)